### PR TITLE
Refactoring JSON for temp files, accepting new Vale template1

### PR DESCRIPTION
**Pull request by rohennes**
WIP

### CODE REVIEW

File: modules/private-clusters-about-gcp.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ The internal load balancer relies on instance groups rather than the target pool
 
 * The cluster IP address is internal only.
 * One forwarding rule manages both the Kubernetes API and machine config server ports.
-* The backend service is comprised of each zone's instance group and, while it exists, the bootstrap instance group.
+* The backend service is comprised of each zone's instance group and, while it exists, the bootstrap instance group.TRIGGER VALE ALERT.
```
Comment (rohennes) [inline modules/private-clusters-about-gcp.adoc:26]: [error] RedHat.TermsErrors: Use 'consist of' rather than 'comprised of'.

Comment (rohennes) [inline modules/private-clusters-about-gcp.adoc:26]: [error] RedHat.TermsErrors: Use 'consist of' rather than 'comprised of'.

Comment (rohennes) [inline modules/private-clusters-about-gcp.adoc:26]: [error] RedHat.Spacing: Keep one space between words in 'p.T'.

Comment (rohennes) [inline modules/private-clusters-about-gcp.adoc:26]: [error] RedHat.TermsErrors: Use 'consist of' rather than 'comprised of'.

Comment (rohennes) [inline modules/private-clusters-about-gcp.adoc:26]: [error] RedHat.Spacing: Keep one space between words in 'p.T'.

File: scripts/function_check_if_existing_comment.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+# Define the check_if_existing_comment function
+function check_if_existing_comment {
+    echo "Checking if existing comment..."
+
+    local vale_json="$1"
+    #local pull_comment_json="$2"
+
+    # Outer loop to iterate through Vale alert JSON
+    echo "$vale_json" | jq -c '.[]' | while read object; do
```
Comment (aireilly) [inline scripts/function_check_if_existing_comment.sh:11]: If we're passing valid json, don't need the '.[]' I think

```suggestion
    echo "$vale_json" | jq | while read object; do
```

**Pull request closed without merging**